distribution/changelog.txt

```diff
@@ -24,7 +24,7 @@
 - Fix: [#12881] Guests' favourite rides are not listed in the guest window.
 - Fix: [#12910] Plugin API: getRide sometimes returns null for valid ride IDs.
 - Fix: [#12912] Plugin: selectedCell of CustomListView is being ignored on creation.
-- Fix: [#12918] Cannot place "Blue Hurricane" (hypercoaster).
+- Fix: [#12918] Cannot place vanilla TD6 tracks of the Hypercoasters, Monster Trucks, Classic Mini Roller Coasters, Spinning Wild Mouses and Hyper-Twisters types.
```

Comment:
When listing out track types, each track type should be listed as singular and not plural.
`- Fix: [#12918] Cannot place vanilla TD6 tracks of the Hypercoaster, Monster Truck, Classic Mini Roller Coaster, Spinning Wild Mouse and Hyper-Twister types.`